benchmarking-cli/src/utils_program.rs

```diff
@@ -0,0 +1,48 @@
+use miden_stdlib::StdLibrary;
+use miden_vm::{Assembler, Kernel, Program, ProgramInfo};
+
+pub struct MidenProgram {
+    pub assembler: Assembler,
+    pub masm_code: String,
+    pub debug: bool,
+    pub program: Option<Program>,
+    pub program_info: Option<ProgramInfo>,
+}
+
+pub const DEBUG_OFF: bool = false;
+//pub const DEBUG_ON: bool = true;
+
+impl MidenProgram {
+    pub fn new(code_as_str: &str, debug: bool) -> Self {
+        Self {
+            assembler: Assembler::default(),
+            masm_code: code_as_str.to_string(),
+            debug,
+            program: None,
+            program_info: None,
+        }
+    }
```

Comment:
It's nice to support both `&str` and `String`. This could be done with:

```suggestion
    pub fn new<S: AsRef<str>>(code_as_str: S, debug: bool) -> Self {
        Self {
            assembler: Assembler::default(),
            masm_code: code_as_str.as_ref().to_string(),
            debug,
            program: None,
            program_info: None,
        }
    }
```

Reply:
this is nice and abstract